cms/models_test.py

```diff
@@ -914,11 +952,40 @@ def test_external_program_page_properties():
     assert external_program_page.description == "<p>desc</p>"
     assert external_program_page.catalog_details == "<p>catalog desc</p>"
     assert external_program_page.duration == "1 week"
+    assert external_program_page.format == FORMAT_OTHER
     assert external_program_page.video_title == "<p>title</p>"
     assert external_program_page.video_url == "http://test.com/mock.mp4"
     assert external_program_page.background_image.title == "background-image"
 
 
+def test_program_page_format_field_default_value():
+    """
+    Verifies that the "format" field in a program page has the default value FORMAT_ONLINE.
+    """
+    program_page = ProgramPageFactory.create()
+
+    assert program_page.format == FORMAT_ONLINE
+
+
+@pytest.mark.parametrize("format_value", [FORMAT_ONLINE, FORMAT_OTHER])
+def test_program_page_format_field_values(format_value):
+    """
+    Verifies that the "format" field in a course page has the default value FORMAT_ONLINE
+    and same context.
+    """
+    program_page = ProgramPageFactory.create(format=format_value)
+    staff_user = StaffUserFactory.create()

```

Comment:
Same as the above comment, Please use `staff_user` fixture.

Reply:
Addressed.